website/docs/security/security-hardening.md

```diff
@@ -47,3 +47,28 @@ To prevent any user from creating/editing CAPTCHA stages block API requests to t
 -   `/api/v3/managed/blueprints*`
 
 With these restrictions in place, CAPTCHA stages can only be edited using [Blueprints on the file system](../customize/blueprints/index.md#storage---file).
+
+### Content Security Policy (CSP)
+
+:::caution
+Setting up CSP incorrectly might result in the client not loading necessary third-party code.
+:::
+
+Content Security Policy (CSP) is a security standard that mitigates the risk of content injection vulnerabilities. authentik doesn't currently support CSP natively, so setting it up depends on your installation. We recommend using a [reverse proxy](../install-config/reverse-proxy.md) to set a CSP header.
+
```

Comment:
We probably should cross-reference https://github.com/goauthentik/authentik/pull/12092 so that that behaviour is also documented. People deploying a custom csp might overlook this and override those headers.

Reply:
Added a general warning not to overwrite any existing CSP header.